recipe/meta.yaml

```diff
@@ -31,8 +31,8 @@ requirements:
     - tornado
     - requests
     - jsonschema
-    - pyqt
-    - pyqtgraph
+    - pyqt 4.11.*
+    - pyqt 4.11.*graph
```

Comment:
The pinning script is clearly not doing the right thing there! It uses `dep.startswith(name)` and that messes with other dependencies. It should be an exact match IMO. Just close this. I will try to fix the script as soon as possible.